### PR TITLE
Convenience script to run different tools on source files

### PR DESCRIPTION
**Pull request by approximator**
- To format code
   `./tools/dev.py run yapf`

 - To run lint checks
   `./tools/dev.py run lint`

(refs #6)

### CODE REVIEW

**Comment by AndreyShelest** [comment]
@approximator , Is it possible to use you scripts without just for installing dependencies? I want just run it once from IDE with already configured virtualenv.

File: tools/dev.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        files_to_format = get_file_list_in(os.path.join(REPO_DIR, 'PocketBudgetTracker'))
+        files_to_format.extend(get_file_list_in(os.path.join(REPO_DIR, 'tools')))
+        for src_file in files_to_format:
+            logger.info("Formatting file {}".format(src_file))
+            run_command("yapf --style='{{based_on_style: google, column_limit: 120}}' -i {}".format(src_file))
+
+    elif args.what_to_run == 'lint':
+        logger.info('Running pylint...')
+        run_command("pylint --version")
+        run_command("pylint --rcfile=.pylintrc PocketBudgetTracker")
```
Comment (AndreyShelest) [inline tools/dev.py]: A lot of hardcoded strings here. "PocketBudgetTracker"

Comment (AndreyShelest) [reply tools/dev.py]: If this is auxiliary script it shouldn't depend on the certain folder name.

Comment (approximator) [reply tools/dev.py]: Fixed

**Comment by approximator** [comment]
> @approximator , Is it possible to use you scripts without just for installing dependencies? I want just run it once from IDE with already configured virtualenv.

Yep

**Comment by approximator** [comment]
see #18

**Pull request closed without merging**